multinet/api/tests/test_upload_csv.py

```diff
@@ -17,24 +17,23 @@
     s3_file_field_re,
     workspace_re,
 )
+from multinet.api.tests.utils import get_field_value
+from multinet.api.utils.workspace_permissions import WorkspacePermission
 
 data_dir = pathlib.Path(__file__).parent / 'data'
 
 
 @pytest.fixture
-def airports_csv(owned_workspace: Workspace, authenticated_api_client, s3ff_client) -> Dict:
-    data_file = data_dir / 'airports.csv'
-    with open(data_file) as file_stream:
-        field_value = s3ff_client.upload_file(
-            file_stream,  # This can be any file-like object
-            data_file.name,
-            'api.Upload.blob',  # The "<app>.<model>.<field>" to upload to
-        )['field_value']
+def airports_csv(workspace: Workspace, user: User, authenticated_api_client, s3ff_client) -> Dict:
+
+    workspace.set_owner(user)
```

Comment:
Since we just want `airports.csv` in the workspace and don't really care how, I think it's best to give the user `writer` permission at the start of the fixture, do the upload, and then remove the `writer` permission before returning from the fixture, as to not interfere with any tests that use this fixture. Also, I'm not sure why there's an extra line here:
```suggestion
    workspace.set_owner(user)
```